src/VisualStudio/Core/Def/SampleToolWindow/DocSymbol.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Imaging.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices
+{
+    using SymbolKind = LanguageServer.Protocol.SymbolKind;
+
+    internal class DocSymbol
+    {
+        public string Name { get; set; }
```

Comment:
I assume we only ever set these once so they can just be `{ get; }` instead of `{ get; set; }`

Reply:
This applies to all the properties in this type right?

Reply:
Yes, it would apply to all properties except for IsExpanded, IsSelected, and Children. This change has been made.